From: =?utf-8?q?Bart=C5=82omiej_Piotrowski?= <bpiotrowski@gnome.org>
Date: Wed, 9 Feb 2022 10:54:17 +0100
Subject: tests/virtme-run: Disable pdcm

There are issues when running a nested KVM with Fedora 35 when pdcm is
enabled. Work around this for now by disabling it.

Related: https://bugzilla.redhat.com/show_bug.cgi?id=2022075
Part-of: <https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/2152>
Origin: upstream, 42.1, commit:7435bfd0e5e0fec7ffe7becb69297e52bd46a91d
---
 src/tests/kvm/virtme-run.sh | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/tests/kvm/virtme-run.sh b/src/tests/kvm/virtme-run.sh
index 3b9fda63..f7631da 100755
--- a/src/tests/kvm/virtme-run.sh
+++ b/src/tests/kvm/virtme-run.sh
@@ -31,7 +31,8 @@ virtme-run \
   --rw \
   --pwd \
   --kimg "$IMAGE" \
-  --script-sh "sh -c \"env $VIRTME_ENV $DIRNAME/run-kvm-test.sh \\\"$WRAPPER\\\" \\\"$WRAPPER_ARGS\\\" \\\"$TEST_EXECUTABLE\\\" \\\"$TEST_RESULT_FILE\\\"\""
+  --script-sh "sh -c \"env $VIRTME_ENV $DIRNAME/run-kvm-test.sh \\\"$WRAPPER\\\" \\\"$WRAPPER_ARGS\\\" \\\"$TEST_EXECUTABLE\\\" \\\"$TEST_RESULT_FILE\\\"\"" \
+  --qemu-opts -cpu host,pdcm=off
 
 TEST_RESULT="$(cat "$TEST_RESULT_FILE")"
 rm "$TEST_RESULT_FILE"
